torch/csrc/jit/ir.cpp

```diff
@@ -96,44 +96,42 @@ std::ostream& operator<<(
   return out;
 }
 
-template <typename T>
-static void printPrimList(std::ostream& out, const std::vector<T>& items) {
-  out << "[";
-  int i = 0;
-  for (auto& item : items) {
-    if (i++ > 0) {
-      out << ", ";
+static void printAttribute(std::ostream& out, const at::Tensor& tensor) {
```

Comment:
The previous conversation was resolved by why do we need `printAttribute` instead of `IValue`'s `<<` print operator/`repr`? Even the non-`AttributeKind::ival` values can just be boxed in a `IValue` before printing

Reply:
We're printing attributes in a specific way that is different than our general ivalue printing - the tensors `{Tensors}` thing